Update tools.mdx (#64)
huggingface/agents-course: units/en/unit1/tools.mdx

When we pass the previous string as part of the input to the LLM, the model will recognize it as a tool, and will know what it needs to pass as inputs and what to expect from the output.

> If you want to provide additional tools, you have to be consistent and always use the same format. The process can be fragile and LLMs may forget some details.

Reviewer: I meant it as "we", as in humans, when we write the instructions manually. Would you like to rephrase it differently?

Author: Ahh okay my bad, I confused it with LLM. 
I would think to rephrase it as:
```
If we want to provide additional tools, we must be consistent and always use the same format. This process can be fragile, and we might accidentally overlook some details.
```

> If you want to provide additional tools, you have to be consistent and always use the same format. The process can be fragile and LLMs may forget some details.

Reviewer: ```suggestion
If we want to provide additional tools, we must be consistent and always use the same format. This process can be fragile, and we might accidentally overlook some details.
```
